pkg/clientinfo/diagnostics.go

```diff
@@ -118,6 +124,32 @@ func (r *Registry) RegisterClientInfoSource(
 	})
 }
 
+// RegisterChainInfo registers the diagnostics source providing
+// information about chains.
+func (r *Registry) RegisterChainInfo(
+	blockCounter chain.BlockCounter,
+) {
+	r.RegisterDiagnosticSource("chain_info", func() string {
+		currentBlock, err := blockCounter.CurrentBlock()
+
```

Comment:
Let's drop the empty line.

Reply:
[0493b72](https://github.com/keep-network/keep-core/pull/3637/commits/0493b72b3f8e4c41684037eaa534beef7433474c)